new audit log content (#1345)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/audit-log.md

---
title: "Audit Log for dbt Cloud Enterprise"
id: audit-log-enterprise
description: "Review actions in your organization for troubleshooting and security."
sidebar_label: "Audit log"
---

> dbt Cloud’s Audit Log allows you to quickly review the actions performed by members of your organization. It includes details such as who performed the action, what the action was, and when it was performed. These details can be used in various troubleshooting and security scenarios.

Author: @saimaddali I'm curious about what kind of troubleshooting and security scenarios are easier to deal with using audit logging?

---
title: "Using the Audit Log for dbt Cloud Enterprise"
id: audit-log
description: "You can review actions in your organization, which can help troubleshoot issues and secure organization."
sidebar_label: "Using the Audit log"
---

dbt Cloud’s Audit Log allows you to quickly review the actions performed by members of your organization. It includes details such as who performed the action, what the action was, and when it was performed. These details can be used in various troubleshooting and security scenarios.

## Accessing the Audit Log

To access Audit Log, click **Account Settings** and **Audit Log**

> <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-menu.png" title="Click Audit log in the Account settings menu."/>

Author: @JKarlavige Is there any way to edit the size of this images inline?

Reviewer: It's not possible to add inline styles directly to the Lightbox component currently, as it doesn't accept a prop for inline styles. 

However, one way to accomplish this is to wrap a Lightbox component with a div, and add the inline styles there. An example of that would be:

```
<div style={{maxWidth: '200px'}}>
  <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-menu.png" title="Click Audit log in the Account settings menu."/>
</div>
```

Author: Thank you Jason! I added these for all of the images.

---
title: "Using the Audit Log for dbt Cloud Enterprise"
id: audit-log

> description: "You can review actions in your organization, which can help troubleshoot issues and secure organization."

Author: ```suggestion
description: "You can troubleshoot possible issues and provide security audits by reviewing event activity in your organization."
```

sidebar_label: "Using the Audit log"
---

To review actions performed by people in your organization, dbt provides logs of audited user and system events. The dbt Cloud Audit log lists events triggered in your organization within the last 90 days.

Use the Audit log to quickly review the actions performed by members of your organization. The Audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. You must be an Org Owner to access the Audit log.

## Accessing the Audit log

To access Audit Log, click **Account Settings** and **Audit Log**.

> <div style={{maxWidth: '150px'}}>

Author: ```suggestion

<div style={{maxWidth: '150px'}}>
```

---
title: "Using the Audit Log for dbt Cloud Enterprise"
id: audit-log
description: "You can troubleshoot possible issues and provide security audits by reviewing event activity in your organization."
sidebar_label: "Using the Audit log"
---

To review actions performed by people in your organization, dbt provides logs of audited user and system events. The dbt Cloud Audit log lists events triggered in your organization within the last 90 days.

Use the Audit log to quickly review the actions performed by members of your organization. The Audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. You must be an Org Owner to access the Audit log.

## Accessing the Audit log

To access Audit Log, click **Account Settings** and **Audit Log**.

<div style={{maxWidth: '150px'}}>

> <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-menu.png" title="Audit log menu"/>

Author: ```suggestion

<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-menu.png" title="Audit log menu"/>

```

| Repository Added   | v1.events.repository.Added   | New repository added        |
| Repository Changed | v1.events.repository.Changed | Repository settings changed |
| Repository Removed | v1.events.repository.Removed | Repository removed          |

### Credentials

| Event Name                       | Event Type                    | Description                      |
| -------------------------------- | ----------------------------- | -------------------------------- |
| Credentials Added to Project     | v1.events.credentials.Added   | Project credentials added        |
| Credentials Changed in Project   | v1.events.credentials.Changed | Credentials changed in project   |
| Credentials Removed from Project | v1.events.credentials.Removed | Credentials removed from project |

## Searching the Audit Log

You can search the Audit log to find a specific event or actor, which is limited to the ones listed in [Events in Audit log](#events-in-audit-log). The Audit log successfully lists historical events spanning the last 90 days. You can search for an actor or event using the search bar, and then narrow your results using the time window.

<div style={{maxWidth: '600px'}}>

> <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-search.png" title="Use search bar to find content in the audit log"/>

Author: ```suggestion

<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-search.png" title="Use search bar to find content in the audit log"/>

```

</div>

## Exporting Logs

You can use Audit log to export historical audit results for security, compliance, and analysis purposes. You can export data for up to the last 90 days. Click the **Export CSV** button to download a CSV file of all the events that occurred in your org over the last two weeks.

<div style={{maxWidth: '200px'}}>

> <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-export-csv.png" title="Download a CSV file"/>

Author: ```suggestion

<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-export-csv.png" title="Download a CSV file"/>

```

---
title: "Using the Audit Log for dbt Cloud Enterprise"
id: audit-log
description: "You can troubleshoot possible issues and provide security audits by reviewing event activity in your organization."
sidebar_label: "Using the Audit log"
---

To review actions performed by people in your organization, dbt provides logs of audited user and system events. The dbt Cloud Audit log lists events triggered in your organization within the last 90 days.

> Use the Audit log to quickly review the actions performed by members of your organization. The Audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. You must be an Org Owner to access the Audit log.

Author: ```suggestion
Use the Audit log to quickly review the actions performed by members of your organization. The Audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. 

You must be an **Account Owner** to access the Audit log and this feature is only available on Enterprise plans.
```

## Accessing the Audit log

To access Audit Log, click **Account Settings** and **Audit Log**.

> <div style={{maxWidth: '150px'}}>

Author: @JKarlavige I added spacing and the images appear large still. Any ideas?

Reviewer: @runleonarun This should now be fixed. It looks like the .docImage class which wraps the image had a width of 600px. I added a max-width for this class to prevent the image from being wider than the div wrapping the lightbox.

Author: Thank you Jason! They are way more reasonably-sized now ☺️

| Repository Removed | v1.events.repository.Removed | Repository removed          |

### Credentials

| Event Name                       | Event Type                    | Description                      |
| -------------------------------- | ----------------------------- | -------------------------------- |
| Credentials Added to Project     | v1.events.credentials.Added   | Project credentials added        |
| Credentials Changed in Project   | v1.events.credentials.Changed | Credentials changed in project   |
| Credentials Removed from Project | v1.events.credentials.Removed | Credentials removed from project |

## Searching the Audit Log

You can search the Audit log to find a specific event or actor, which is limited to the ones listed in [Events in Audit log](#events-in-audit-log). The Audit log successfully lists historical events spanning the last 90 days. You can search for an actor or event using the search bar, and then narrow your results using the time window.

<div style={{maxWidth: '600px'}}>

<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-search.png" title="Use search bar to find content in the audit log"/>

</div>

## Exporting Logs

> You can use Audit log to export historical audit results for security, compliance, and analysis purposes. You can export data for up to the last 90 days. Click the **Export CSV** button to download a CSV file of all the events that occurred in your org over the last two weeks.

Author: ```suggestion
You can use Audit log to export historical audit results for security, compliance, and analysis purposes. You can export data for up to the last 90 days. Click the **Export CSV** button to download a CSV file of all the events that occurred in your org over the last 90 days.
```

---
title: "Using the Audit Log for dbt Cloud Enterprise"
id: audit-log
description: "You can troubleshoot possible issues and provide security audits by reviewing event activity in your organization."
sidebar_label: "Using the Audit log"
---

To review actions performed by people in your organization, dbt provides logs of audited user and system events. The dbt Cloud Audit log lists events triggered in your organization within the last 90 days.

Use the Audit log to quickly review the actions performed by members of your organization. The Audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. 

> You must be an **Account Owner** to access the Audit log and this feature is only available on Enterprise plans.

Author: ```suggestion
You must be an **Account Admin** to access the Audit log and this feature is only available on Enterprise plans.
```

| Repository Removed | v1.events.repository.Removed | Repository removed          |

### Credentials

| Event Name                       | Event Type                    | Description                      |
| -------------------------------- | ----------------------------- | -------------------------------- |
| Credentials Added to Project     | v1.events.credentials.Added   | Project credentials added        |
| Credentials Changed in Project   | v1.events.credentials.Changed | Credentials changed in project   |
| Credentials Removed from Project | v1.events.credentials.Removed | Credentials removed from project |

## Searching the Audit Log

You can search the Audit log to find a specific event or actor, which is limited to the ones listed in [Events in Audit log](#events-in-audit-log). The Audit log successfully lists historical events spanning the last 90 days. You can search for an actor or event using the search bar, and then narrow your results using the time window.

<div style={{maxWidth: '600px'}}>

<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-search.png" title="Use search bar to find content in the audit log"/>

</div>

## Exporting Logs

> You can use Audit log to export historical audit results for security, compliance, and analysis purposes. You can export data for up to the last 90 days. Click the **Export CSV** button to download a CSV file of all the events that occurred in your org over the last 90 days.

Author: ```suggestion
You can use Audit log to export historical audit results for security, compliance, and analysis purposes. You can export data for up to the last 90 days. Click the **Export CSV** button to download a CSV file of all the events that occurred in your organization over the last 90 days.
```
